### PR TITLE
Issue1704 move nrpm mover to obsolete

### LINKED ISSUE #1704: Move Nrpm mover model to the Obsolete package
First discussed in https://github.com/ibpsa/modelica-ibpsa/pull/1646#issuecomment-1404315591.

This issue is to move `IBPSA.Fluid.Movers.SpeedControlled_Nrpm` (and `.Preconfigured.SpeedControlled_Nrpm`) to `IBPSA.Obsolete`.
Reasons behind this:
- About 33 lines of code are copied between `IBPSA.Fluid.Movers.{SpeedControlled_Nrpm,SpeedControlled_y}`. This makes the maintenance of these two models cumbersome and error-prone. (As a note, the duplication between the two flow-controlled models is about 13 lines.)
- Introducing another layer of inheritance to absorb these copied lines would probably make the `Movers` package even more complex and difficult to maintain for future developers.
- We (LBNL) have no knowledge of user actually using the `Nrpm` model.

For the reasons above, it makes sense to move the `Nrpm` mover model to the `Obsolete` package and get it out of the way.

I will leave this issue open for three weeks to leave time for feedback on this proposed change before working on the implementation.

### PR DESCRIPTION
**Pull request by hcasperfu**
This closes #1704.

Changes made in this PR:
- `Movers.SpeedControlled_Nrpm` and `Movers.Preconfigured.SpeedControlled_Nrpm` are moved to `Obsolete`.
- In the following models, any and all `_Nrpm` mover models are replaced with `_y` models.
  - `Validation.PowerSimplified`, and its extended models `PowerExact` and `PowerEuler`. The components on the canvas are also moved lower to make space.
  - `Validation.Pump_Nrpm_stratos`, and it is renamed as `Pump_y_stratos`.
  - `Validation.Pump_stratos`.
  - `Validation.PumpCurveDerivatives`.
- The model `Validation.SpeedControlled_Nrpm` is moved to `Obsolete`, because it specifically tests the `_Nrpm` model and a `_y` equivalent of it already exists.
- The following models are copied to `Obsolete`, and the `_Nrpm` mover in the original models is deleted:
  - Models that extend from `Validation.BaseClasses.ControlledFlowMachine`:
    - `Validation.ControlledFlowMachine`
    - `Validation.ControlledFlowMachineDynamic`
    - `Preconfigured.Validation.ControlledFlowMachinePreconfigured`
  - `Examples.MoverParameter`, and its extended models `MoverContinuous` and `MoverStages`.

### CODE REVIEW

File: IBPSA/Obsolete/Fluid/Movers/SpeedControlled_Nrpm.mo
Comment (mwetter) [inline IBPSA/Obsolete/Fluid/Movers/SpeedControlled_Nrpm.mo:1]: This model misses the `obsolete` annotation, see https://github.com/ibpsa/modelica-ibpsa/blob/040a3d1c09f9fa523d5fc518c3aa1b73e66de292/IBPSA/Obsolete/Fluid/FixedResistances/PlugFlowPipe.mo#L215

Without it, users won't get a warning.

Also, it needs to extend from `IBPSA.Obsolete.BaseClasses.ObsoleteModel`
https://github.com/ibpsa/modelica-ibpsa/blob/040a3d1c09f9fa523d5fc518c3aa1b73e66de292/IBPSA/Obsolete/Fluid/FixedResistances/PlugFlowPipe.mo#L3

**Review by mwetter — CHANGES_REQUESTED** [review]
Please see the inline comment. Also, please search for `rpm` in `IBPSA.Movers` as there are various references that should be removed. We should formulate the model and documentation without any reference to rpm as rpm is no longer used. Otherwise, users may think the rpm values have some relevance.

Also, links such as 
```
<a href=\"modelica://IBPSA.Fluid.Movers.FlowControlled_Nrpm\">
IBPSA.Fluid.Movers.FlowControlled_Nrpm</a>
```
need to be changed as this will lead to an invalid link in the generated html.

File: IBPSA/Obsolete/Fluid/Movers/SpeedControlled_Nrpm.mo
Comment (hcasperfu) [reply IBPSA/Obsolete/Fluid/Movers/SpeedControlled_Nrpm.mo:1]: Question: Once I mark the model with an obsolete annotation, some validation/example models that use this component no longer simulate. I see that the `Obsolete` package is not in IBPSA's CI tests. So do I just leave them broken?

Comment (mwetter) [reply IBPSA/Obsolete/Fluid/Movers/SpeedControlled_Nrpm.mo:1]: @hcasperfu : Models marked as obsolete should still simulate and work in the CI tests (they do for `Buildings`). But you are right, `IBPSA/.travis.yml` does not include `IBPSA.Obsolete`. This is a mistake and it should be added.

File: IBPSA/Fluid/Movers/Data/Generic.mo
[the remark this answers is not in the document]
Comment (mwetter) [reply IBPSA/Fluid/Movers/Data/Generic.mo:1]: Looks good, except for one missed paragraph that I deleted in 0c17ba76c

**Review by mwetter — APPROVED** [review]

**Comment by mwetter** [comment]
This is ready to merge, but we should have one non-LBNL reviewer.

@hcasperfu : can you please merge this downstream to the `Buildings` library `master` (not the `maint_9.1.x` branch).

**Review by Mathadon — APPROVED** [review]
that's a serious cleanup, thanks!

**Pull request merged**